app/assets/stylesheets/alchemy/errors.scss

```diff
@@ -37,3 +37,10 @@ div#errors, div.errors {
     margin-bottom: 0.5em;
   }
 }
+
+body.error {
+  .error.message {
+    max-height: 95%;
+    overflow-y: scroll;
+  }
+}
```

Comment:
I was actually hoping that we can directly change the error message's classes to give this specific `error.message` an additional class, say `scroll` to assign it the additional css styling with that.